pubnet/core/etc/stellar-core.cfg

```diff
@@ -7,6 +7,8 @@ DATABASE="postgresql://dbname=core host=localhost user=stellar password=__PGPASS
 NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"
 CATCHUP_RECENT=100
 
+EXPERIMENTAL_BUCKETLIST_DB=true
```

Comment:
Should we update this after publishing stable core? Wondering if someone wants to test protocol 20 now and might encounter a core start up failure with the new flag.

Reply:
It shouldn't fail, should it? We probably should have switched a while ago, and doing this now decouples the changes a bit (bucketlist DB now, v21 build later)

Reply:
Since the core_ref in [build-latest](https://github.com/dmkozh/quickstart/blob/17bacf728bd4c316ed00c11ef32a167cad166d3c/Makefile#L28) is pointing to 20.4.0, core will complain during startup while parsing this [config flag](https://github.com/stellar/stellar-core/blob/c6f474133738ae5f6d11b07963ca841909210273/src/main/Config.cpp#L1657).

Reply:
The flag has been there for a while (https://github.com/stellar/stellar-core/blob/c6f474133738ae5f6d11b07963ca841909210273/src/main/Config.cpp#L1084) and our pubnet validators have been running with it for more than two weeks now.

Reply:
I meant the other way round when a flag is passed but the core image is old then there will be a parsing error. In case someone in the ecosystem builds the docker quickstart image with `stellar/quickstart:latest` option, they will get core v20.4.0 and when they trying joining mainnet network they will see this error.

Reply:
`EXPERIMENTAL_BUCKETLIST_DB` was available in v20 I thought?